packages/plugins/Pins/README.md

```diff
@@ -17,7 +17,7 @@ The usage of `displayComponent` has no influence on the creation of Pins on the
 | appearOnClick    | appearOnClick | Pin restrictions. See object description below.                                                                                                                                                                                                                                                     |
 | coordinateSource | string        | The pins plugin may react to changes in other plugins. This specifies the path to such store positions. The position must, when subscribed to, return a GeoJSON feature. Please mind that, when referencing another plugin, that plugin must be in `addPlugins` before this one. |
 | style            | style?        | Display style configuration.                                                                                                                                                                                                                                                                        |
-| initial          | number[]?                               | Configuration options for setting an initial pin.                                                                                                                                                                                                                                                   |
+| initial          | initial? | Configuration options for setting an initial pin.                                                                                                                                                                                                                                                   |
```

Comment:
The indentation of the third column should be the same as with the other table rows.